client/telemetry/src/lib.rs

```diff
@@ -287,38 +311,46 @@ impl Stream for Telemetry {
 	}
 }
 
-impl slog::Drain for TelemetryDrain {
-	type Ok = ();
-	type Err = ();
-
-	fn log(&self, record: &slog::Record, values: &slog::OwnedKVList) -> Result<Self::Ok, Self::Err> {
-		let before = Instant::now();
-
-		let serialized = async_record::AsyncRecord::from(record, values);
-		// Note: interestingly, `try_send` requires a `&mut` because it modifies some internal value, while `clone()`
-		// is lock-free.
-		if let Err(err) = self.sender.clone().try_send(serialized) {
-			warn!(target: "telemetry", "Ignored telemetry message because of error on channel: {:?}", err);
-		}
-
-		if before.elapsed() > Duration::from_millis(50) {
-			warn!(target: "telemetry", "Writing a telemetry log took more than 50ms");
-		}
-
-		Ok(())
-	}
-}
-
+/// TODO doc
 /// Translates to `slog_scope::info`, but contains an additional verbosity
 /// parameter which the log record is tagged with. Additionally the verbosity
 /// parameter is added to the record as a key-value pair.
-#[macro_export]
+#[macro_export(local_inner_macros)]
 macro_rules! telemetry {
-	 ( $a:expr; $b:expr; $( $t:tt )* ) => {
-		$crate::with_logger(|l| {
-			$crate::slog::slog_info!(l, #$a, $b; $($t)* )
-		})
-	}
+	( $a:expr; $b:expr; $( $t:tt )* ) => {{
+		let message_verbosity: u8 = $a;
+		let mut json = format_fields_to_json!($($t)*);
+		json.insert("level".into(), "INFO".into());
```

Comment:
This probably still needs to be changed?

Reply:
No, that's the original behavior. Here is the original code:

https://github.com/paritytech/substrate/blob/d5464dd654cade80dc00cc723e2dcbb6d56d86d3/client/telemetry/src/lib.rs#L319

It was using `slog_info!` so this was **always** INFO.

Reply:
Weird. What is the telemetry doing with this? In the end it seems like redundant information that are not required?

Reply:
@maciejhirsz maybe you can help us here? Is this level needed at all? It was/is always INFO

Reply:
Telemetry isn't doing anything with the log level, we could drop it, just needs to be dropped from the backend first else it will reject incoming messages.

Reply:
This has been dropped.